### PR TITLE
Milestone: Use GitHub Pages to deploy my portfolio project.

### PR DESCRIPTION
**Pull request by oliverSCZ**
I did the following:

- Created a new branch, 'add-live-demo' 
- Added all files required and followed the necessary instructions (news screenshots).
- Committed files (as required).
- Effectively set up a HTML & CSS linter.
- Documented my work professionally and added **a live demo link** in README.md file to the project's repository
- Opened this pull request as instructed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,23 @@
 > This is my showcase of my recently works
 
 ![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot1.png)
+![screenshot](./app_screenshot2.png)
+![screenshot](./app_screenshot3.png)
+![screenshot](./app_screenshot4.png)
+![screenshot](./app_screenshot5.png)
```
Comment (D4nielJ) [inline README.md:12]: - [ ] Let's add the screenshots to the PR as well, as they appear right now as broken links. But first, read my comment below.

[OPTIONAL] I think that only one meaningful screenshot it's enough. If the visitors of your repo are interested in the project they will visit the live demo link.

File: README.md [hunk 2]
```diff
@@ -5,18 +5,23 @@
...
-- HtML file
+- HTML file
 - CSS file
-- linter
+- Linter
 
 ## Live Demo
 
-[Live Demo Link](https://oliverscz.github.io/mobile-version-project/)
+[Live Demo Link](https://oliverscz.github.io/mobile-version-project/)
```
Comment (D4nielJ) [inline README.md:24]: [OPTIONAL] The links of your navbar are currently linking to pages that don't exist and generate a 404 error. Either use a placeholder (#) until you develop this feature. Or link them to the proper sections of your page. You can do this by assigning id's to section tags and linking the href to the proper id (e.g. #about, #works, etc.)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @oliverSCZ, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: 

- Good job following GitHub flow and passing all of the linters.
- You successfully did use of GitHub pages feature! :rocket:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @oliverSCZ, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oliverSCZ** [comment]
Thank you guys, your words motivate me to continue. Cheers and Happy coding!👏👏👏

**Pull request merged**